packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -300,14 +300,19 @@ library LibProving {
             }
 
             if (tid == 1 && tran.prover == blk.assignedProver) {
-                // For the first transition, if the previous prover is still the
-                // assigned prover, we exclusively grant permission to the
-                // assigned approver to re-prove the block, unless the proof
-                // window has elapsed.
+                // For the first transition, (1.) if the previous prover is
+                // still the assigned prover, we exclusively grant permission to
```

Comment:
a reminder to change line 257

Reply:
> a reminder to change line 257

To be honest, no i do not think so. We needed at line line 117, but here (line 257), it is like "reProposed" so the time re-starts. I think it is fine as is.